common/interface.go

```diff
@@ -289,6 +289,12 @@ type EnableEpochsHandler interface {
 	RefactorPeersMiniBlocksEnableEpoch() uint32
 	GetCurrentEpoch() uint32
 
+	IsFlagDefined(flag core.EnableEpochFlag) bool
```

Comment:
will you remove the ...EnableEpoch() and IsFlag...() methods in a subsequent PR?

Reply:
exactly, once all of them are replaced with calls to new ones